### DISCUSSION TITLE
Feature request: support `hashicorp/google` and `hashicorp/google-beta` v6

### DISCUSSION THREAD

**Discussion by kkom** [GoogleCloudPlatform/cluster-toolkit#1909] [Ideas and feature requests]
As in the title – it would be great if you could support v5 of the Google Terraform provider. Right now the version constraints resolve to `< 5.0.0`, e.g. see this:

https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/e64f027e3d90717506a3610c52038438f7442526/modules/file-system/filestore/versions.tf#L19-L22

We're already on v5 for all of our other Terraform workspaces, and it would let us have a consistent version across all of our infra.

Best wishes,
Konrad

**Comment by harshthakkar01** [comment]
Hi,

We have identified the changes that we need to make and have this in our roadmap.

**Reply by kkom** [reply]
Thanks @harshthakkar01 !

**Reply by kkom** [reply]
Hey @harshthakkar01 – just a quick note that v6 is out now!

https://github.com/hashicorp/terraform-provider-google/releases/tag/v6.0.0
